src/cli/uenv.cpp

```diff
@@ -23,7 +26,9 @@ int main(int argc, char** argv) {
 
     CLI11_PARSE(cli, argc, argv);
 
-    fmt::print(fmt::emphasis::bold | fg(fmt::color::orange), "{}\n", settings);
+    uenv::init_log();
```

Comment:
To do: this should take the `--verbose` flag into account. Probably update the log level wherever the verbose flag is handled.

Reply:
https://github.com/bcumming/uenv2/pull/3/commits/1578fb6e60a0f7309a4bc237616236fe4a8446b3